Doc 4856 fgac (#16691)
cockroachdb/docs: _includes/cockroachcloud/cockroachcloud-ask-admin.md

{{site.data.alerts.callout_info}}

> Only [Console Admins](console-access-management.html#console-admin) can create SQL users. If you are a [Developer](console-access-management.html#developer), you need to ask your Console Admin for the credentials of a SQL user to access the cluster. To find out who your Console Admin is, [log in](https://cockroachlabs.cloud/) and navigate to **Cluster Overview** > **Access**.

Author: Hmm, good point. That was old copy. Is there a way to find out who your Admin is?
If you are just an org member, can you see the /access page and see who has what roles so you can identify an admin?

> Only [Console Admins](console-access-management.html#console-admin) can create SQL users. If you are a [Developer](console-access-management.html#developer), you need to ask your Console Admin for the credentials of a SQL user to access the cluster. To find out who your Console Admin is, [log in](https://cockroachlabs.cloud/) and navigate to **Cluster Overview** > **Access**.

Reviewer: If you're an org member, there isn't a way right now. For others, they can go to the access page. I believe we're planning on opening the `/access` page up to org members in phase 2 so that everyone can see the admins, but that's still tbd.

> Only [Console Admins](console-access-management.html#console-admin) can create SQL users. If you are a [Developer](console-access-management.html#developer), you need to ask your Console Admin for the credentials of a SQL user to access the cluster. To find out who your Console Admin is, [log in](https://cockroachlabs.cloud/) and navigate to **Cluster Overview** > **Access**.

Author: so if you are an org member, is there any way to 'find out who your org admin is'? Does it say in your invitation to the org who invited you?

> Only [org admins](authorization.html#org-administrator-legacy) can create SQL users. If you are a [Developer](authorization.html#org-developer-legacy), you need to ask your org admin for the credentials of a SQL user to access the cluster. To find out who your org admin is, [log in](https://cockroachlabs.cloud/) and navigate to **Cluster Overview** > **Access**.

Author: ```suggestion
Only [org admins](authorization.html#org-administrator-legacy) can create SQL users. If you are a [Developer](authorization.html#org-developer-legacy), you need to ask your org admin for the credentials of a SQL user to access the cluster. You can find the email of a cluster administrator for your cluster from the console's **Access** page, at `https://cockroachlabs.cloud/access`.
```

Reviewer 2: Is it worth linking "Access page" to https://www.cockroachlabs.com/docs/cockroachcloud/console-access-management.html?

Author: That page has been removed, and it's content absorbed into managing-access.md
